resources/core/charts/application-connector/charts/metadata-service/values.yaml

```diff
@@ -19,5 +19,5 @@ minioUserSecretName: remoteenvironments-minio-user
 tests:
   image:
     name: metadata-service-tests
-    tag: 0.3.280
+    tag: 09bf23c0
```

Comment:
Please update also image path for tests in `templates/tests/test-acceptance.yaml` to use the `develop` directory.